README.md

```diff
@@ -60,52 +60,54 @@ You can remove permissions you don't use:
    ```ruby
    post_install do |installer|
      installer.pods_project.targets.each do |target|
+       flutter_additional_ios_build_settings(target)
+
        target.build_configurations.each do |config|
          ... # Here are some configurations automatically generated by flutter
-   
+
          # You can remove unused permissions here
-         # for more infomation: https://github.com/BaseflowIT/flutter-permission-handler/blob/develop/permission_handler/ios/Classes/PermissionHandlerEnums.h
+         # for more information: https://github.com/BaseflowIT/flutter-permission-handler/blob/develop/permission_handler/ios/Classes/PermissionHandlerEnums.h
```

Comment:
This link should start with: https://github.com/Baseflow/...

(So not "BaseflowIT")

Reply:
Good catch, changed it